src/ontology/mondo-edit.obo

```diff
@@ -440183,7 +440184,7 @@ xref: ICD10CM:Q79.6 {source="Orphanet:99875/attributed", source="Orphanet:99875/
 xref: Orphanet:99875 {source="MONDO:equivalentObsolete"}
 xref: UMLS:C0268345 {source="Orphanet:99875", source="MONDO:notFoundInDiseaseSubset"}
 xref: UMLS:C3508773 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
-is_a: MONDO:0007525 {source="Orphanet:99875"} ! Ehlers-Danlos syndrome, arthrochalasis type
+is_a: MONDO:0007525 {source="Orphanet:99875"} ! Ehlers-Danlos syndrome, arthrochalasia type
```

Comment:
for some reason this change is coming through on all my PRs today. I see it on master. It probably won't resolve until my PRs are merged but it can be ignored.